app/src/main/java/lt/vilnius/tvarkau/fragments/NewReportFragment.kt

```diff
@@ -552,7 +556,7 @@ class NewReportFragment : BaseFragment(),
     }
 
     override fun hideProgress() {
-        progressDialog?.hide()
+        progressDialog?.dismiss()
     }
 
     override fun onDestroyView() {
```

Comment:
Also there is no need to set it to null `progressDialog = null` if you are using `progressDialog::dismiss`